src/theme.js

```diff
@@ -0,0 +1,15 @@
+const colors = {
+  white: '#fff',
+  black: '#000',
+  darkgray: '#222',
+  lightgray: '#eee',
+  silver: '#c4c4c4',
+  gray: '#999',
+  yellow: '#ffd25d',
+};
+
+const theme = {
+  colors,
+};
+
+export default theme;
```

Comment:
fontSize 도 관리 하는 것도 좋습니다 :) 요건 취향차이!
BCSDLab 에서 디자인 템플릿 있다면 참고하면 좋을듯합니다

```suggestion
const fontSize = {
  sm: '12px',
  md: '18px',
  lg: '24px',
}

const theme = {
  colors,
  fontSize
};

export default theme;
```